src/intrprtr.c

```diff
@@ -3457,14 +3451,9 @@ void            IntrUnbList ( Int narg )
 */
 void            IntrElmList ( Int narg )
 {
-  Obj                 elm = (Obj) 0;            /* element, result                 */
```

Comment:
Could the 0 be there because some compiler complains about returning an unassigned value? (which would happen unless the compiler knows that `SyntaxError` doesn't return).